ports/demonstone64/port.json

```diff
@@ -1,29 +1,30 @@
 {
-    "version": 2,
-    "name": "demonstone64.zip",
-    "items": [
-        "Demonstone64.sh",
-        "demonstone64"
+  "version": 3,
+  "name": "demonstone64.zip",
+  "items": [
+    "Demonstone64.sh",
```

Comment:
in gameainfo.xml you wrote ./Demonstone 64.sh
Here it is without space. make sure it is uniform